Dockerfile

```diff
@@ -1,12 +1,12 @@
 # based on https://github.com/nodejs/docker-node/blob/master/4.7/slim/Dockerfile
 
-FROM node:10
+FROM node:13.3
```

Comment:
used this chart to choose a version that has the npm version needed. Decided to just try the latest version first.